test/sql/test_agg_function/T/test_mann_whitney

```diff
@@ -0,0 +1,31 @@
+-- name: testMannWhitney
+select 
+    mann_whitney_u_test(x, t, 'two-sided') 
+from 
+    TABLE(generate_series(1, 10)) as numbers(x), TABLE(generate_series(0, 1)) as idx(t);
+
+select 
+    mann_whitney_u_test(x, t, 'less') 
+from 
+    TABLE(generate_series(1, 10)) as numbers(x), TABLE(generate_series(0, 1)) as idx(t);
+
+select 
+    mann_whitney_u_test(x, t, 'greater') 
+from 
+    TABLE(generate_series(1, 10)) as numbers(x), TABLE(generate_series(0, 1)) as idx(t);
+
+
+select 
+    mann_whitney_u_test(x, t, 'two-sided') 
+from 
+    TABLE(generate_series(1000, 10000)) as numbers(x), TABLE(generate_series(0, 1)) as idx(t);
+
+select 
+    mann_whitney_u_test(x, t, 'less') 
+from 
+    TABLE(generate_series(1000, 10000)) as numbers(x), TABLE(generate_series(0, 1)) as idx(t);
+
+select 
+    mann_whitney_u_test(x, t, 'greater') 
+from 
+    TABLE(generate_series(1000, 10000)) as numbers(x), TABLE(generate_series(0, 1)) as idx(t);
```

Comment:
Can you add more tests by using a real Starrocks table since there may be some differences(multi stage aggs) for real table ?
```
CREATE TABLE t1 (
    c1 int,
    c2 boolean,
    c3 tinyint,
    c4 int,
    c5 bigint,
    c6 largeint,
    c7 string
    )
DUPLICATE KEY(c1)
DISTRIBUTED BY HASH(c1) BUCKETS 3
PROPERTIES ("replication_num" = "1");

INSERT INTO t1 values
    (1, true, 11, 111, 1111, 11111, "111111"),
    (2, false, 22, 222, 2222, 22222, "222222"),
    (3, true, 33, 333, 3333, 33333, "333333"),
    (4, null, null, null, null, null, null),
    (5, -1, -11, -111, -1111, -11111, "-111111"),
    (6, null, null, null, null, "36893488147419103232", "680564733841876926926749214863536422912");

```